helpers/calendar/ratios_and_utilization.py

```diff
@@ -149,14 +149,14 @@ def map_results_to_ratio_class(self,
             )
         return result
 
-    def percentage_formater(portion, total):
+    def percentage_formater(self, portion, total):
         """ Calculates the percentage of the entered portion to the total and returns it
          :params
             - portion, total
         """
         try:
             percentage = (portion/total) * 100
-            return round(percentage, 1)
+            return round(percentage, 2)
```

Comment:
Hey @RonKbS, Nice work, I have a small concern, the task is requiring us to be as accurate as possible, Is rounding off to two decimal places the best we can go for or we can go ahead and round off to 3 or 4. Or you could include clear screenshots that show the differences between those two results other than showing the difference in the decimal places. This will show more about the accuracy of 2 decimal places.

Reply:
I have gone with removing the removing the round function as recommended by the TC @nakatuddesuzan , @kampkelly and @caveinn.

Tests are failing now because I have included a check for calender_id to make sure data is not corrupted by creating rooms with similar calender_ids. This is because the id is used in analytics to retrieve data for rooms

Reply:
Hey @RonKbS, isn't this out of scope?

Reply:
I'm seeing it as being part of 'increasing the accuracy of the cancellation percentage'.

The story's description also refers to bookings. I, therefore, thought to add the calender_id check as another means of improving the accuracy because in my case, for example, the percentages returned on running the analytics were similar. This was because the rooms I created had the same calender_id.